samples/tutorial/src/Binding/HandlerBindingSample.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Threading.Tasks;
+using static Binding.GetInstanceSample;
+
+namespace Binding
+{
+    internal static class HandlerBindingSample
+    {
+        internal static async Task<int> MultipleArgs()
+        {
+            #region MultipleArgs
+
+            var command = new RootCommand
+            {
+                new Option("--a-string") { Argument = new Argument<string>() },
+                new Option("--an-int") { Argument = new Argument<int>() }
+            };
+
+            command.Handler = CommandHandler.Create(
```

Comment:
I wonder if this would be more clear with a method, even a local method, than the lambda

Reply:
I was opting for an older construct on the assumption more people would be familiar with it.

Reply:
FWIW, we tend to prefer using the newer features where applicable. It encourages adoption for readers to see them.

When I read these samples, the lambda "feels" natural to me. In production code, would you use a lambda or a local (even local static) function? I don't know if seeing the lambda is because it looks familiar, or if it's "right".

Reply:
As to which is right, it depends, of course. :smile: I suppose for a smaller codebase the lambda seems simpler and more cohesive, but for larger codebases the static or local function helps break the code into more understandable pieces. In `dotnet try` our handlers are static methods in separate classes entirely. Among other things, this enables testing them directly.